main/integration-tests/support/setup.js

```diff
@@ -106,7 +111,7 @@ class Setup {
     const session = await getClientSession({ idToken, setup: this });
     this.sessions.push(session);
 
-    return session;
+    return { session, username };
```

Comment:
Do we need this? I though `<session>.resources.currentUser.get()` also provides the username.

Reply:
Nevermind, I see how you're using it.